platform/getting-started-with-particular-service-platform.md

```diff
@@ -8,6 +8,8 @@ tags:
 - Visual Studio
 ---
 
+include: sm-discontinued
```

Comment:
Ah, you meant `SM directory`. I was wondering about the am! Yeah, I agree. It's just that I looked for usage of Matrix and disabled it. But will moving it cause a 404?